src/main/java/at/ac/tuwien/kr/alpha/common/rule/head/Head.java

```diff
@@ -0,0 +1,21 @@
+package at.ac.tuwien.kr.alpha.common.rule.head;
+
+/**
+ * Represents the head of a rule, i.e., either a choice or a disjunction of atoms, but not both. For normal rules the disjunction contains only one atom.
+ *
+ * Copyright (c) 2017-2019, the Alpha Team.
+ */
+public abstract class Head {
+
+	public abstract boolean isNormal();
```

Comment:
I agree it's strange, but the actual problem here lies a bit deeper.. Ideally, every head would be instantiated with the correct type, i.e. a head that is normal would also be a `NormalHead` from the start. However, that's not what's happening - the parser makes a `DisjunctiveHead` out of everything that's not a `ChoiceHead`. That leads to scenarios where we need to cast from `DisjunctiveHead` to `NormalHead`, and before doing that cast we need to check if the respective `DisjunctiveHead` is actually a normal one. Renaming the method on `DisjunctiveHead` would in my opinion only mask the real problem.
I removed all obsolete occurrences of the `isNormal` check now, taking care of those cases where normality is ensured by the type system anyway.
I created issue #267 to modify `ProgramParser` as mentioned above since I'd rather not bloat this PR any more.